nixos/modules/services/web-apps/lemmy.nix

```diff
@@ -71,6 +85,10 @@ in
 
   config =
     lib.mkIf cfg.enable {
+      warnings = lib.optional (cfg.settings.federation.enabled) ''
```

Comment:
Accessing the option works, because it still exists. Ideally, we could remove the option entirely and use `attryByPath` for this check. If that doesn't work, the option should be made hidden.

Reply:
For removing the option I opened https://github.com/NixOS/nixpkgs/pull/237471; I'll look into how to hide this then I guess.

Reply:
Okay I think [this recent force-push](https://github.com/NixOS/nixpkgs/compare/a8058c7b4c7bc445e1c8b1c23e470d565dacd640..a3527512d7a9fbad6a20aab6eacfcae101028757) should hide it. Kinda hacks, not sure if that's the intended way to do this.

Reply:
so the idea if I understand correctly is to backport this visible = false and to remove it completly on unstable.

Reply:
Yep